.github/workflows/build.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
         with:
           fetch-tags: true
```

Comment:
Should these be `fetch-depth: 0`?

Reply:
I opted for fetching the tags as part of the `incrementVersion` step. That allowed me to leave the fetch-depth at 1 (default) and remove `fetch-tags` since it's not respected anyways.